example.env

```
@@ -66,6 +66,7 @@ MANAGER_DASHBOARD_FIREBASE_MESSAGING_SENDER_ID=
 MANAGER_DASHBOARD_FIREBASE_APP_ID=
 MANAGER_DASHBOARD_SENTRY_DSN=
 MANAGER_DASHBOARD_SENTRY_TRACES_SAMPLE_RATE=
+MANAGER_DASHBOARD_COMMUNITY_DASHBOARD_URL=https://community-dashboard.mapswipe.org
```

```suggestion
MANAGER_DASHBOARD_COMMUNITY_DASHBOARD_URL=
```

Should we have values on example.env?

---

Yes.